doc/faq.rst

```diff
@@ -61,6 +61,92 @@ reporting a bug, which should look something like this::
     skcuda:        Not found
 
 
+Why shouldn't I "pickle" my MNE-Python objects for later use?
+-------------------------------------------------------------
+`Pickling <https://docs.python.org/3/library/pickle.html>`_ data and
+MNE-Python objects for later use can be tempting due to its simplicity
+and generality, but it is usually not the best option:
+
+- http://www.benfrederickson.com/dont-pickle-your-data/
+- http://stackoverflow.com/questions/21752259/python-why-pickle
+
```

Comment:
"short it is not made for stable persistence and it is very likely that you will not be able to read your data in a future not too far from the moment you saved them." or so.